requirements.txt

```diff
@@ -7,7 +7,7 @@ canonicalwebteam.yaml-responses==1.1.1
 canonicalwebteam.image-template==1.0.0
```

Comment:
yaml-responses on the previous line should also be removed as it's included in flask-base:

https://github.com/canonical-web-and-design/canonicalwebteam.flask-base/blob/master/canonicalwebteam/flask_base/app.py#L62

Reply:
Done!